packages/wrangler/src/__tests__/generate.test.ts

```diff
@@ -43,7 +43,7 @@ describe("generate", () => {
 				`"✨ Created no-template/wrangler.toml"`
 			);
 			expect(std.warn).toMatchInlineSnapshot(`
-			"[33m▲ [43;33m[[43;30mWARNING[43;33m][0m [1mThe \`init\` command is no longer supported. Please use \`mockpm create cloudflare@2 no-template\` instead.[0m
+			"[33m▲ [43;33m[[43;30mWARNING[43;33m][0m [1mThe \`init\` command is no longer supported. Please use \`mockpm create cloudflare/@2 no-template\` instead.[0m
```

Comment:
something weird is going on with jest snapshot encoding
- double backslash `\\` is changed into a forward slash `/`
- this doesn't effect real usage: `\\@` is what is actually returned from shell-quote and that works

You can confirm by running `wrangler init` and seeing the C3 command logged out correctly